doc/user/content/sql/drop-materialized-view.md

```diff
@@ -1,20 +1,14 @@
 ---
 title: "DROP MATERIALIZED VIEW"
-description: "`DROP MATERIALIZED VIEW` removes a materialized view from your Materialize instances."
+description: "`DROP MATERIALIZED VIEW` removes a materialized view from Materialize."
 menu:
   main:
     parent: commands
 ---
 
-`DROP MATERIALIZED VIEW` removes a materialized view from your Materialize instances.
-
-## Conceptual framework
-
-Materialize maintains materialized views after you create them. If you no longer need the
-materialized view, you can remove it.
-
-Because materialized views rely on receiving data from sources, you must drop all materialized views that
-rely on a source before you can [drop the source](../drop-source) itself. You can achieve this easily using **DROP SOURCE...CASCADE**.
+`DROP MATERIALIZED VIEW` removes a materialized view from Materialize. If there
+are other views depending on the materialized view, you must explicitly drop
+them first, or use the `CASCADE` option.
```

Comment:
Trying to grok why we allow these two scenarios:

- **Dropping a materialized view from a cluster that is not the cluster maintaining the view:** is this supposed to be addressed by RBAC rather than catalog dependencies?
- **Dropping a materialized view that is indexed in the active cluster, or in other clusters:** is it right that only other views are considered dependencies of materialized views? Is the idea that indexes, sinks or subscriptions that depend upon a dropped materialized view should live on, but simply not get updated from that point on?

Reply:
> Dropping a materialized view from a cluster that is not the cluster maintaining the view: is this supposed to be addressed by RBAC rather than catalog dependencies?

Yes, exactly. The active cluster plays no role in `DROP` behavior. (Except maybe to prevent you from dropping the currently-active cluster? I forget what our behavior is there.)

> Dropping a materialized view that is indexed in the active cluster, or in other clusters: is it right that only other views are considered dependencies of materialized views? Is the idea that indexes, sinks or subscriptions that depend upon a dropped materialized view should live on, but simply not get updated from that point on?

Sinks should also prevent the drop of a materialized view. Tails are treated as ephemeral (e.g., they don't survive an `environmentd` restart) so they _don't_ block the drop of a materialized view (or a normal view, for that matter). Indexes don't block the drop of a materialized view because, idk, that's just how they work on tables in other SQL databases.

Reply:
Thanks for clarifying, @benesch!